app/schemas/subject_create_schema.rb

```diff
@@ -5,6 +5,10 @@ class SubjectCreateSchema < JsonSchema
     required "links", "locations"
     additional_properties false
 
+    property "external_id" do
```

Comment:
I figured it's just a nice thing to do to expose it, since it's also serialized back? But don't care too much, no thought was really going into this on my part.

Reply:
Hmm, well there's no check to make sure that `external_id` remains unique within a subject set. I guess with this added to the normal API you could make it so that on a subsequent Import it's no longer specified which subject it'll upsert. Then again, using `update_links` you could add existing subjects to a set and create clashes as well. 

Do you think it's worth it to add stuff to enforce uniqueness (either in Rails or PG)? It'll be a non-trivial query, and we don't really expect people that use Imports to also do other uploading into that same set. 

Alternatively, we could mark the subject set as "using_imports" and disallow normal API linking?

Reply:
I don't think we will need it, let's just be aware of this if issues arise.